src/cleanlab_codex/internal/validator.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import annotations
+
+from typing import Any
+
+from pydantic import BaseModel, Field
+
+from cleanlab_codex.utils.errors import MissingDependencyError
+
+try:
+    from cleanlab_tlm.utils.rag import Eval, TrustworthyRAGScore, get_default_evals
+except ImportError as e:
+    raise MissingDependencyError(
+        import_name=e.name or "cleanlab-tlm",
+        package_url="https://github.com/cleanlab/cleanlab-tlm",
+    ) from e
+
+
+"""Evaluation metrics (excluding trustworthiness) that are used to determine if a response is bad."""
+EVAL_METRICS = ["response_helpfulness"]
+
+"""Evaluation metrics that are used to determine if a response is bad."""
+BAD_RESPONSE_EVAL_METRICS = ["trustworthiness", *EVAL_METRICS]
+
+
+class IsBadResponseConfig(BaseModel):
```

Comment:
```suggestion
class BadResponseThreshold(BaseModel):
```

Reply:
Yes, moving it to the user-facing module.